src/browser/Page.zig

```diff
@@ -907,29 +907,41 @@ pub fn createElement(self: *Page, ns_: ?[]const u8, name: []const u8, attribute_
         break :blk .html;
     };
 
+    var name = name_;
+    var ns_prefix_: ?[]const u8 = null;
+    if (std.mem.indexOfPos(u8, name_, 0, ":")) |pos| {
```

Comment:
indexOfScalarPos with a `':'` avoids 1 length check...but, see above comment about whether this is correct for an HTML document.